packages/payload/src/admin/components/forms/field-types/Relationship/select-components/MultiValueLabel/index.tsx

```diff
@@ -24,7 +24,8 @@ export const MultiValueLabel: React.FC<MultiValueProps<Option>> = (props) => {
         draggableProps,
         // @ts-expect-error // TODO Fix this - moduleResolution 16 breaks our declare module
         setDrawerIsOpen,
-        // onSave,
+        // @ts-expect-error // TODO Fix this - moduleResolution 16 breaks our declare module
+        onSave,
```

Comment:
If I remember correctly, the reason this was commented out was because it does not work across both monomorphic and polymorphic relationships (or something similar to that). I.e. when you close the drawer, the new value should be automatically populated in the select input with correct title. I don't believe this was fully working after switching to `react-select` v5.